web/src/installers/index.ts

```diff
@@ -456,6 +456,7 @@ export class Installer {
       "0.9.11",
       "0.9.10",
       "0.9.9",
+      "alpha",
```

Comment:
@areed is this ok? how will this affect things?

Reply:
That's fine. Users will be able to specify alpha in their yaml spec and alpha will show as the last option on kurl.sh.